terraform/paas/statuscake.tf

```diff
@@ -0,0 +1,14 @@
+resource "statuscake_test" "alert" {
```

Comment:
Defaults values can be removed, see other comment

Reply:
Changed to be the same as above